### ISSUE TITLE
Remove LICENSE and yarn.lock example

### ISSUE THREAD

**Issue by jeroentvb** [FlafyDev/create-spicetify-app#2]
Aforementioned files are included when downloading the repo with `create-spicetify-app` and I think they shouldn't be for the following reasons:

#### LICENSE
Users should choose their own LICENSE. Right now it comes with the MIT license, with you as the copyright holder. I'm not an expert on this, but doesn't that make you the owner of every project generated?

#### yarn.lock
I think it's a good idea to run `npn install` or `yarn` after downloading/generating the project files, so the lockfile matches the users preferred package manager. For example, this package uses the yarn.lock from spicetify-creator, while I use npm, which will generate a package-lock.json.

**Comment by FlafyDev** [comment]
You're right, I haven't thought about all the other files that are being downloaded from the repo.
I'm going to create a new branch here to fit Spicetify Creator's new branch "package" (a package instead of `./creator/`, https://github.com/FlafyDev/spicetify-creator/issues/2 ).
So based on what you said, I think the process should go like this:
1. create a folder for the project.
2. generate a `package.json` with the necessary scripts (like build) and dependencies.
3. generate a tsconfig
4. do `npm install` or `yarn` based on user input.

Let me know if you have a better way to do it because I'm not entirely sure how that would go yet.

**Comment by jeroentvb** [comment]
Sound good. A few additions:
* For a better overview I would turn this repo into a monorepo with `create-spicetify-app` and `spicetify-creator` meaning those packages would be in the same branch, but separate folders.
* `.gitignore` and `README.md` would also be good to be included in the generate files script

**Comment by FlafyDev** [comment]
I agree a Monorepo will make things way easier. But wouldn't it make more sense to turn the `spicetify-creator` repo into the Monorepo? (Because of its name)

**Comment by jeroentvb** [comment]
Yeah, that may be a better idea.
